cogs/event_handler.py

```diff
@@ -210,7 +210,7 @@ async def on_member_remove(self, member: nextcord.Member):
         with suppress(nextcord.errors.HTTPException):
             embed = nextcord.Embed(
                 color=0x0DD91A,
-                title=f"Bye {member.display_name} :wave:\nIt was great having you!",
+                title=f"Bye {member.display_name} :wave:\nIt was great having you! We hope to see you back soon.",
```

Comment:
It would be nice to have the extra text on a new line

Reply:
how do i do that?

Reply:
@codeman1o1